packages/api/jest.config.js

```diff
@@ -1 +1,5 @@
-module.exports = require('../../jest.config.base');
+const baseConfigForPackages = require('../../jest.config.base');
```

Comment:
These 3 files for jest were added here because of a system issue: `No tests found, exiting with code 1`. It'll be fixed separately in Farzin PR, but it was necessary to include it here too in order to run tests.